### PR TITLE
fix: Viewer on public album

### PR DESCRIPTION
**Pull request by y-lohse**
Any idea for relevant tests? Shallow-rendering snapshot gives:

```
<div>
  <Main
    className="u-pt-1-half"
  >
    <Selection />
    <IconSprite />
  </Main>
</div>
```

…which is pretty useless. And I don't think deep-rendered snapshot are very useful to catch regressions.

### CODE REVIEW

**Comment by y-lohse** [comment]
[Photos - visionneuse album photos vue publique](https://trello.com/c/u5Uyeu31/1689-photos-visionneuse-album-photos-vue-publique)

File: src/photos/targets/public/App.jsx [hunk 1]
```diff
@@ -44,8 +44,17 @@ class App extends Component {
       .downloadArchive(allPhotos.map(({ _id }) => _id), album.name)
   }
 
+  renderViewer(children) {
```
Comment (Crash--) [inline src/photos/targets/public/App.jsx]: Can we add a comment to explain why and when we can have `children` ?

**Comment by cedricmessiant** [comment]
About the test, I'm not sure you can do something relevant as the code stays as it is. 
Maybe a deep rendered snapshot is better than nothing though (it's our responsibility to update it carefully).

**Comment by y-lohse** [comment]
Here we go, added some tests. The snapshots are still pretty big but it's not a full render either.

File: src/photos/targets/public/App.jsx [hunk 2]
```diff
@@ -114,6 +126,9 @@ App.propTypes = {
   photos: PropTypes.array.isRequired,
   fetchMore: PropTypes.func.isRequired
 }
+App.contextTypes = {
+  t: PropTypes.func
```
Comment (Crash--) [inline src/photos/targets/public/App.jsx]: 👍

**Pull request merged**